test/test.js

```diff
@@ -12,7 +12,8 @@ require('bedrock-kms-http');
 require('bedrock-passport');
 require('bedrock-permission');
 require('bedrock-profile');
-require('bedrock-profile-http');
+// require('bedrock-profile-http');
+require('..');
```

Comment:
The worst part is this.  Although we could continue adding the module to the test package file with the `file:..` syntax, we would require it using a relative require.

I'm going to spend a bit more time attempting to see if nyc can be bent to our will here.  But I'm not hopeful due to the nyc issue referenced in the PR description.